tests/round_robin_pmap.cpp

```diff
@@ -0,0 +1,112 @@
+/*
+ *  This file is a part of TiledArray.
+ *  Copyright (C) 2013  Virginia Tech
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "TiledArray/pmap/round_robin_pmap.h"
+#include "global_fixture.h"
+#include "tiledarray.h"
+#include "unit_test_config.h"
+
+using namespace TiledArray;
+
+struct RoundRobinPmapFixture {
+  RoundRobinPmapFixture() {}
+};
+
+// =============================================================================
+// RoundRobinPmap Test Suite
+
+BOOST_FIXTURE_TEST_SUITE(round_robin_pmap_suite, RoundRobinPmapFixture)
+
+BOOST_AUTO_TEST_CASE(constructor) {
+        for (std::size_t tiles = 1ul; tiles < 100ul; ++tiles) {
```

Comment:
needs clang-formatting ... also install git-hooks as described in https://valeevgroup.github.io/tiledarray/dox-master/_workflow-_notes.html so this does not happen in the future